src/common/support.js

```diff
@@ -249,6 +249,17 @@ function getSupportInfo(version, opts) {
       }
 
       return newOption;
+    })
+    .map(option => {
+      const filteredPlugins = plugins.filter(
+        plugin => plugin.defaultOptions && plugin.defaultOptions[option.name]
+      );
+      const pluginDefaults = filteredPlugins.reduce((reduced, plugin) => {
+        const printerName = Object.keys(plugin.printers)[0];
+        reduced[printerName] = plugin.defaultOptions[option.name];
```

Comment:
We probably need to attach the plugin name (package name, plugin filepath, or the `name` property if provided) in `load-plugins.js` and use it here as an ID.

https://github.com/prettier/prettier/blob/9643d3370233f2870c43b81d0855833fb983d9f4/src/common/load-plugins.js#L6